azerothcore/azerothcore-wotlk
# DB\Quest: Mage Summoner

##### CHANGES PROPOSED:

-  Added gossip menu option for Shaman Trainer
-  Updated faction for creature Minor Water Guardian
-  Updated flags for creature Sarilus Foulborne
-  Added SAI for Creature Sarilus Foulborne


###### ISSUES ADDRESSED:


Closes 


##### TESTS PERFORMED:

Tested in game, the quest is working fine now and can be done.


##### HOW TO TEST THE CHANGES:

1. For missing Shaman Trainer ( .go creature 4664 ) and speak with NPC
2. There should be '' Train '' option.
3. For Quest Mage Summoner, Use Alliance character and ( .quest add 1017 )
4. .go creature 15084 and fight it 
5. The Creature should cast Spells and Summons Guardians


##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
